node/jaeger/src/spans.rs

```diff
@@ -124,6 +124,13 @@ impl PerLeafSpan {
 	}
 }
 
+/// Default implementation, can be handy in tests dependent upon a mocked PerLeafSpan.
```

Comment:
Do we actually use it now ?

Reply:
Yes. One of the tests invokes a method which takes a span as an argument, so just added `default()` so the test don't break.